packages/addons/sveltekit-adapter/index.ts

```diff
@@ -0,0 +1,100 @@
+import { defineAddon, defineAddonOptions } from '@sveltejs/cli-core';
+import { exports, functions, imports, object, type AstTypes } from '@sveltejs/cli-core/js';
+import { parseJson, parseScript } from '@sveltejs/cli-core/parsers';
+
+type Adapter = {
+	id: string;
+	package: string;
+	version: string;
+};
+
+const adapters: Adapter[] = [
+	{ id: 'node', package: '@sveltejs/adapter-node', version: '^5.2.9' },
+	{ id: 'static', package: '@sveltejs/adapter-static', version: '^3.0.6' },
+	{ id: 'vercel', package: '@sveltejs/adapter-vercel', version: '^5.5.0' },
+	{ id: 'cloudflare-pages', package: '@sveltejs/adapter-cloudflare', version: '^4.8.0' },
+	{ id: 'cloudflare-workers', package: '@sveltejs/adapter-cloudflare-workers', version: '^2.6.0' },
+	{ id: 'netlify', package: '@sveltejs/adapter-netlify', version: '^4.4.0' }
+];
+
+const options = defineAddonOptions({
+	adapter: {
+		type: 'select',
+		question: 'Which SvelteKit adapter would you like to use?',
+		options: adapters.map((p) => ({ value: p.id, label: p.id, hint: p.package })),
+		default: 'node'
+	}
+});
+
+export default defineAddon({
+	id: 'sveltekit-adapter',
```

Comment:
```suggestion
	alias: 'adapter',
```
Should we add an alias?

Reply:
yea, I wasn't sure if this was a good idea or not. I'm not completely against it though. I _don't_ think we'll have another `X-adapter` add-on anyway, so we could probably get away with it